src/pal/src/cruntime/printfcpp.cpp

```diff
@@ -1145,6 +1146,8 @@ int CoreVfwprintf(CPalThread *pthrCurrent, PAL_FILE *stream, const wchar_16 *for
     PERF_ENTRY(vfwprintf);
     ENTRY("vfwprintf (stream=%p, format=%p (%S))\n",
           stream, format, format);
+    
+    va_copy(ap, aparg);
```

Comment:
I am still not sure I understand the need for the addition of the va_copy here. The stackoverflow article you have referred to talks about *build* errors on amd64 that they are solving using this. Did you get build errors on ARM or did it have some runtime issues?